### ISSUE TITLE
Github account that is empty

### ISSUE THREAD

**Issue by ghost** [kousen/Spring-Framework-Essentials#3]
This link has no source from OReilly, maybe update?
https://github.com/oreillymedia/Learning-Path-Learn-Spring-and-Spring-Boot

**Comment by kousen** [comment]
Yes, that's clearly wrong. It should be https://github.com/kousen/spring-and-spring-boot . I'll try to figure out who to notify to update that.
